gui/main.cpp

```diff
@@ -24,20 +31,61 @@ int main(int argc, char *argv[]) {
   parser.addOption(startupOnlyOption);
   parser.process(app);
 
-  auto generate_data = []() -> std::vector<std::tuple<float, float>> {
+  GuiStateContainer state_container(
+      /*history_window=*/std::chrono::milliseconds(30000));
+  auto startup_time = state_container.GetStartupTime();
+
+  std::unique_ptr<ConnectedDevice> device =
+      std::make_unique<FakeConnectedDevice>(
+          [&](const GuiStatus &gui_status,
+              ControllerStatus *controller_status) {
+            // For now, completely fake the data and ignore gui_status.
+            (void)gui_status;
+
+            controller_status->uptime_ms =
+                std::chrono::duration_cast<std::chrono::milliseconds>(
+                    std::chrono::steady_clock::now() - startup_time)
+                    .count();
```

Comment:
wow it's like they tried to make the std::chrono as ugly as possible.

Reply:
It is truly remarkable. I'll add a header with some typedefs.

Reply:
Added.